packages/nextjs/package.json

```diff
@@ -21,6 +21,8 @@
     "@uniswap/v2-sdk": "^3.0.1",
     "blo": "^1.0.1",
     "daisyui": "4.5.0",
+    "ethereum-blockies-base64": "^1.0.2",
+    "ethers": "v5",
```

Comment:
Lets remove both of this library, we already have good alternatives of this in SE-2 🙌

checkout https://github.com/BuidlGuidl/batch4.buidlguidl.com/pull/23/files#r1560902439